lambda_functions/user_service/create_user.py

```diff
@@ -29,13 +29,16 @@ def create_user(event):
                 GroupName = 'Detective'
             )
 
-            # Add mail address if submitted
-            if 'email' in event['request']['userAttributes']:
+            # Add mail address if submitted and set confirmation status
+            if 'custom:mail_subscription' in event['request']['userAttributes']:
+                confirmation_status = event['request']['userAttributes']['custom:mail_subscription']
                 mail = Mail()
                 mail.email = event['request']['userAttributes']['email']
                 mail.user_id = user.id
+                if confirmation_status == 0:
```

Comment:
Ich glaube, der confirmation status in dem Event ist ein String und keine Zahl. Ich hatte dir auch mal ein Beispiel via Slack geschickt.

Reply:
das hatte ich doch auch beim letzten Mal schon korrigiert:
https://github.com/DetektivKollektiv/python-lambda-functions/blob/c71649725cbecde0333549d0d96bd6083c4d0027/lambda_functions/user_service/create_user.py#L38

Reply:
Achso, stimmt. Komisch, dass mir das dann nochmal so angezeigt wurde. Ich bin eigentlich nur durch die neuesten commits, da sollte es dann eigentlich schon eingebaut sein. Aber vielleicht hab ich auch irgendwas falsch gemacht. Sorry für die Verwirrung und danke fürs Aufklären!